tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -520,6 +542,50 @@ def _test_data_repository_task(remote_command_executor, mount_dir, bucket_name,
     assert_that(file_permissions).is_equal_to("0100777")
 
 
+def _test_storage_capacity(remote_command_executor, mount_dir, storage_capacity):
+    logging.info("Test FSx storage capacity")
+    if storage_capacity:
+        result = remote_command_executor.run_remote_command(
+            f"df -BG | grep {mount_dir}" + "| awk '{print$2}' | tr -d -c 0-9"
+        )
+        assert_that((storage_capacity - int(result.stdout)) // storage_capacity).is_less_than(0.2)
```

Comment:
Might be worth adding some explanation in comments whenever we have hardcoded tolerances like this. Any documentation explaining this? Will this tolerance be the same for all FSx storage type?

Reply:
add a comment to explain it